Jenkinsfile

```diff
@@ -108,7 +108,7 @@ pipeline {
                         cp debian/changelog ${WORKSPACE}/debian/changelog-${SERIES_VERSION}
                         sed -i "s/${PKG_VERSION}/${NEW_PKG_VERSION}/" ${WORKSPACE}/debian/changelog-${SERIES_VERSION}
                         dpkg-source -l${WORKSPACE}/debian/changelog-${SERIES_VERSION} -b .
-                        sbuild --nolog --verbose --dist=${BUILD_SERIES} --no-run-lintian --append-to-version=~${SERIES_VERSION}  ../ubuntu-advantage-tools*${NEW_PKG_VERSION}*dsc
+                        sbuild --resolve-alternatives --nolog --verbose --dist=${BUILD_SERIES} --no-run-lintian --append-to-version=~${SERIES_VERSION}  ../ubuntu-advantage-tools*${NEW_PKG_VERSION}*dsc
```

Comment:
Do we need this --resolve-alternatives anymore now that you have reordered the d/control dep order?

Reply:
AFAICT, you only need `--resolve-alternatives` for building in Groovy and before. Build in Hirsute and onward should pick up debhelper (>= 13.3) and should be good!

However, a small note below..

Reply:
So yep we still need it here!